var/spack/repos/builtin/packages/py-testpath/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyTestpath(PythonPackage):
+    """Testpath is a collection of utilities for Python code working with
+    files and commands."""
+
+    homepage = "https://github.com/jupyter/testpath"
+    url      = "https://pypi.io/packages/source/t/testpath/testpath-0.4.2.tar.gz"
+
+    version('0.4.2', sha256='b694b3d9288dbd81685c5d2e7140b81365d46c29f5db4bc659de5aa6b98780f8')
+
+    depends_on('py-pip', type='build')
+
+    def install(self, spec, prefix):
+        pip = which('pip')
+        pip('install', '.', '--prefix={}'.format(prefix))
```

Comment:
Is this install method and pip dependency actually needed? Pip is usually only needed to install wheels.

Reply:
It failed for me without, but I didn't investigate further, so there might be an option do go without it

Reply:
Post the error message and I’ll take a look.

Reply:
couldn't reproduce.... That makes me wonder what kind of strange setup I had... This OS upgrade was significantly more painful than the last one.